src/components/Resources/AllResources.js

```diff
@@ -1,17 +1,45 @@
 import React, { Component } from 'react';
-import { Link } from "react-router-dom";
+import './AllResources.css'
+import articleImg from '../../assets/articles.jpg'
+import liveImg from '../../assets/liveEvents.jpg'
+import videoImg from '../../assets/video.jpg'
 
 class AllResources extends Component {
```

Comment:
same as last comment, use `function AllResources(props)`

Reply:
Could you show an example please?